downstream/modules/platform/proc-configuring-the-metrics-utility.adoc

```diff
@@ -47,13 +56,24 @@ export METRICS_UTILITY_REPORT_EMAIL="email@email.com"
 export METRICS_UTILITY_REPORT_RHN_LOGIN="test_login"
 export METRICS_UTILITY_REPORT_COMPANY_BUSINESS_LEADER="BUSINESS LEADER"
 export METRICS_UTILITY_REPORT_COMPANY_PROCUREMENT_LEADER="PROCUREMENT LEADER"
+
+# Build the report
+metrics-utility build_report
 ----
 +
-. Run the following command to gather and store the data in the provided `SHIP_PATH` directory:
+. To ensure that these files are executable, run: 
 +
 [source, ]
 ----
-metrics-utility gather_automation_controller_billing_data --ship --until=10m
+chmod a+x /home/my-user/cron-gather 
+/home/my-user/cron-report
```

Comment:
```suggestion
chmod a+x /home/my-user/cron-gather /home/my-user/cron-report
```

should be one line, not two, otherwise we're running the script, not making it runnable